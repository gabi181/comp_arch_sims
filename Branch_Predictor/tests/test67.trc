8 8 2 2 local_history global_tables not_using_share
0x1CD7CB18 N 0x4F149AD8
0x3CCF92F0 N 0x434D5210
0x45B2DAC0 N 0x343B8A28
0x1CD7CB18 T 0x4F149AD8
0x1CD7CB18 N 0x4F149AD8
0x09C06204 N 0x2AC2F4EC
0x1CD7CB18 T 0x4F149AD8
0x45B2DAC0 N 0x343B8A28
0x3CCF92F0 T 0x434D5210
0x1CD7CB18 T 0x4F149AD8
0x09C06204 N 0x2AC2F4EC
0x3CCF92F0 T 0x434D5210
0x3CCF92F0 N 0x434D5210
0x0A92A40C N 0x75570B64
0x45B2DAC0 N 0x343B8A28
0x0A92A40C T 0x75570B64
0x1CD7CB18 T 0x4F149AD8
0x09C06204 T 0x2AC2F4EC
0x1CD7CB18 T 0x4F149AD8
