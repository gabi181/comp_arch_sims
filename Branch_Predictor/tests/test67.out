0x1cd7cb18 N 0x1cd7cb1c
0x3ccf92f0 N 0x3ccf92f4
0x45b2dac0 N 0x45b2dac4
0x1cd7cb18 N 0x1cd7cb1c
0x1cd7cb18 T 0x4f149ad8
0x9c06204 N 0x9c06208
0x1cd7cb18 T 0x4f149ad8
0x45b2dac0 N 0x45b2dac4
0x3ccf92f0 N 0x3ccf92f4
0x1cd7cb18 T 0x4f149ad8
0x9c06204 N 0x9c06208
0x3ccf92f0 N 0x3ccf92f4
0x3ccf92f0 T 0x434d5210
0xa92a40c N 0xa92a410
0x45b2dac0 N 0x45b2dac4
0xa92a40c N 0xa92a410
0x1cd7cb18 T 0x4f149ad8
0x9c06204 N 0x9c06208
0x1cd7cb18 T 0x4f149ad8
flush_num: 7, br_num: 19, size: 840b
